package.json

```diff
@@ -57,14 +57,13 @@
         "re-reselect": "^4.0.0",
         "react-chartjs-2": "^4.1.0",
         "react-final-form": "^6.5.7",
-        "react-query": "4.0.0-alpha.10",
+        "react-query": "^4.0.0-beta.23",
```

Comment:
Note that this means that we now have both `react-query@3` and `react-query@4` in the app. V3 is used by `useDataQuery`, which should only be used by the `OrganisationUnitTree`.